app/api/resources/user.py

```diff
@@ -224,7 +224,7 @@ class VerifiedUser(Resource):
             f"{messages.AUTHORISATION_TOKEN_IS_MISSING['message']}"
         }
     )
-    @users_ns.marshal_list_with(public_user_api_model)  # , skip_none=True
+    @users_ns.marshal_list_with(public_user_api_model, code=HTTPStatus.OK, description="Success")  # , skip_none=True
```

Comment:
@17sushmita why you add status code here? If not done purposely please revert

Reply:
Same for other instances as well

Reply:
I've done it purposely as not mentioning status code in marshal statement was the reason why hardcoded status code was displayed in response table. This statement was automatically generating a response example and taking 200 as status code implicitly.

Reply:
Not sure about this. Pinging @PrashanthPuneriya for review

Reply:
Thanks @17sushmita for your explanation

Reply:
You have done a great job in figuring this out @17sushmita :)